### ISSUE TITLE
tests don't run on OSX

### ISSUE THREAD

**Issue by andreasbaumann** [patrickfrey/strusUtilities#41]
```
The following tests FAILED:
	  1 - TestAnalyzeSubSegmenter1 (Failed)
	  2 - TestAnalyzeSubSegmenter2 (Failed)
	  3 - TestAnalyzeSubSegmenter3 (Failed)
	  4 - TestAnalyzeBase1 (Failed)
	  5 - TestAnalyzeJson1 (Failed)
	  6 - TestAnalyzeJson2 (Failed)
	  7 - TestAnalyzeConfig1 (Failed)
	  8 - TestAnalyzeConfig2 (Failed)
	  9 - TestAnalyzeWithDocType1 (Failed)
	 10 - TestInsertWithDocType1 (Failed)
	 11 - TestInsertBase1 (Failed)
	 12 - TestSimpleQuery1 (Failed)
	 13 - TestSummarization1 (Failed)

12: Test command: /Users/administrator/strus/strusUtilities/tests/scripts/runTest.sh "testSimpleQuery1"
12: Test timeout computed to be: 9.99988e+06
12: /Users/administrator/strus/strusUtilities/tests/scripts
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 20: /Users/administrator/strus/strusUtilities/src/strusCreate/strusCreate: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 44: /Users/administrator/strus/strusUtilities/src/strusInsert/strusInsert: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 53: /Users/administrator/strus/strusUtilities/src/strusQuery/strusQuery: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 53: /Users/administrator/strus/strusUtilities/src/strusQuery/strusQuery: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 53: /Users/administrator/strus/strusUtilities/src/strusQuery/strusQuery: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 53: /Users/administrator/strus/strusUtilities/src/strusQuery/strusQuery: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 53: /Users/administrator/strus/strusUtilities/src/strusQuery/strusQuery: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 53: /Users/administrator/strus/strusUtilities/src/strusQuery/strusQuery: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts/ENV: line 26: /Users/administrator/strus/strusUtilities/src/strusDestroy/strusDestroy: No such file or directory
12: /Users/administrator/strus/strusUtilities/tests/scripts
12: DIFF AT 25

find . -name strusQuery -perm 0755 -type f
./src/strusQuery/Release/strusQuery
```

Windows and XCode builds binaries not in the current directory, but in `Release` or `Debug`
depending on the Cmake project build option switch.

**Comment by andreasbaumann** [comment]
I can really only recommend getting rid of shell-based script frameworks because they cause
more trouble maintaining on the long-term...

**Comment by patrickfrey** [comment]
I temporarily disabled the script tests on OS/X. I currently cannot figure out the correct path to set for the binaries to execute yet.

**Comment by andreasbaumann** [comment]
Fixed in pull request #43

**Comment by andreasbaumann** [comment]
Fixed. Closing bug.
